skrub/_reporting/_data/templates/tabs.html

```diff
@@ -15,23 +15,23 @@
         <div class="tab-list tab-list-scroller" data-manager="TabList" >
             <div>
                 <button type="button" data-target-panel-id="dataframe-sample-panel"
-                    data-role="tab" data-is-selected data-test="sample-tab" class="tab"
+                    data-role="tab" {% if default_panel_id == "dataframe-sample-panel" %}data-is-selected{% endif %} data-test="sample-tab" class="tab"
```

Comment:
```suggestion
                    data-role="tab" {% if open_panel_id == "dataframe-sample-panel" %}data-is-selected{% endif %} data-test="sample-tab" class="tab"
```

As you renamed the name of the variable in the template environment you also need to change it here (and everywhere it's used in the template)